src/core_modules/capture-core/components/WidgetEventNote/WidgetEventNote.epics.ts

```diff
@@ -1,11 +1,12 @@
-// @flow
 import { batchActions } from 'redux-batched-actions';
 import { ofType } from 'redux-observable';
 import { featureAvailable, FEATURES } from 'capture-core-utils';
 import { map, switchMap } from 'rxjs/operators';
 import uuid from 'd2-utilizr/lib/uuid';
 import moment from 'moment';
 import { actionTypes, batchActionTypes, startAddNoteForEvent } from './WidgetEventNote.actions';
+import type { ClientNote, FormNote, SaveContext } from './WidgetEventNote.types';
+import type { ReduxStore, ApiUtils, EpicAction } from 'capture-core-utils/types';
```

Comment:
Please fix the  `Absolute imports should come before relative imports  import/first` eslint error by moving this line at the top of the file